starwars/src/components/Characters.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+class Characters extends React.Component {
+    render() {
+        return this.props.starwarsChars.map((chars) => (
+            <h2 className="char-style">{ chars.name }</h2>
```

Comment:
Mapping over props and returning an element of the data 👍 
You *could* extend componental structure by returning a <Character /> component for every prop, but this works great.